manifests/init.pp

```diff
@@ -164,7 +164,7 @@
 #
 # $agent_noop::                             Run the agent in noop mode.
 #
-# $puppetmaster::                           Hostname of your puppetmaster (server
+# $puppetmaster::                           Hostname of your puppetserver (server
```

Comment:
We may also need to update this, but I don't really know what a good name would be. `agent_server` is also not that clear and just `server` is already taken.

Reply:
maybe `agent_server_hostname`?